swift_browser_ui_frontend/src/components/ShareModal.vue

```diff
@@ -423,15 +431,20 @@ export default {
   }
 
   c-select {
-    color: var(--csc-dark-grey);
+    color: $csc-dark-grey;
   }
 
   c-link > span {
     font-size: 0.875rem;
   }
 
-  .field {
-    margin: 2rem 0 0 0;
+  #shareid-tags {
+    margin: 1rem 0;
+    border: 1px solid $csc-dark-grey;
```

Comment:
Shouldn't this be part of the taginput component?

Reply:
I would love to. But one thing is the `<c-select>` component has a darker border and current it cannot be modified from our repo, so I had to make the Taginput in this ShareModal a bit darker color to match with the `c-select`

Reply:
I see what you mean! Makes me wonder why aren't all the fields looking the same, as they come from `csc-ui`